### PR TITLE
[new release] clarity-lang (0.1.0)

### PR DESCRIPTION
**Pull request by artob**
Clarity smart contract parser and AST

- Project page: <a href="https://github.com/weavery/clarity.ml">https://github.com/weavery/clarity.ml</a>

##### CHANGES:

All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

### Added

- The initial release.

### CODE REVIEW

File: packages/clarity-lang/clarity-lang.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+maintainer: ["Arto Bendiken <arto@bendiken.net>"]
+authors: ["Arto Bendiken <arto@bendiken.net>"]
+license: "Unlicense"
+homepage: "https://github.com/weavery/clarity.ml"
+bug-reports: "https://github.com/weavery/clarity.ml/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "alcotest" {with-test}
+  "menhir" {>= "20200624"}
+  "num" {>= "1.3"}
```
Comment (mseri) [inline packages/clarity-lang/clarity-lang.0.1.0/opam:14]:
```suggestion
  "num" {>= "1.3"}
  "cppo" {build}
```
The cppo package is used for building

Comment (mseri) [reply packages/clarity-lang/clarity-lang.0.1.0/opam:14]: Can you please upstream this change?

Comment (artob) [reply packages/clarity-lang/clarity-lang.0.1.0/opam:14]: @mseri Thanks, indeed needs to be added!

Comment (artob) [reply packages/clarity-lang/clarity-lang.0.1.0/opam:14]: https://github.com/weavery/clarity.ml/commit/ab2c04273fdbce856da7fb55161598e92de5e43d

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**